docs/resources/built-in-decorators.md

```diff
@@ -65,3 +65,32 @@ lint:
       tagNames:
         pet: ./my-custom-description.md
 ```
+
+### hide-internals
+
+Using `x-internal` tag by default to hide nodes:
+```
+  PathItem
+  Operation
+  Schema
+  Response
+  RequestBody
+  Example
+  MediaType
+  Server
+  Callback
+  Parameter
```

Comment:
We are missing `Tag` too. That should be added. 

We are missing Schema property (this is different from hiding an entire schema -- just hiding a specific property).

Reply:
Removing tags won't remove corresponding operations though (and I'm not sure this would be a good idea).

We should probably add a note to the docs because it may be confusing otherwise.

Reply:
No, we also need to remove tags though. Remember, we do render tags with no operations.

Reply:
What do you mean by "we also need to remove tags though"? Remove removed tag from all the operations that use it? Or remove operations? (I don't think we should remove operations)